android/app/src/main/kotlin/com/linagora/android/tmail/MainActivity.kt

```diff
@@ -0,0 +1,11 @@
+package com.linagora.android.tmail
+
+import io.flutter.embedding.android.FlutterFragmentActivity
+import io.flutter.embedding.engine.FlutterEngine
+import io.flutter.plugins.GeneratedPluginRegistrant
+
+class MainActivity: FlutterFragmentActivity() {
```

Comment:
why we need to create this file?

Reply:
run android

Reply:
But you delete another `MainActivity`. Please help me to understand more in one comment, no need to save the time with short sentences.

Reply:
Because I renamed packagegage, git saved it like that.